packages/web3-providers/src/types/Firefly.ts

```diff
@@ -219,6 +222,10 @@ export namespace FireflyRedPacketAPI {
         ens_name: string
     }
 
+    export interface RedPacketClaimedInfoWithNumberChainId extends WithNumberChainId<RedPacketClaimedInfo> {}
+    export interface RedPacketSentInfoWithNumberChainId extends WithNumberChainId<RedPacketSentInfo> {}
+    export interface RedPacketClaimListInfoWithNumberChainId extends WithNumberChainId<RedPacketClaimListInfo> {}
```

Comment:
Use a simpler name outside the provider.

```suggestion
    export interface RedPacketClaimedInfo extends WithNumberChainId<RedPacketClaimedInfoItem> {}
    export interface RedPacketSentInfo extends WithNumberChainId<RedPacketSentInfoItem> {}
    export interface RedPacketClaimListInfo extends WithNumberChainId<RedPacketClaimListInfoItem> {}
```